backend/docker_entrypoint.sh

```diff
@@ -8,15 +8,20 @@ BE_PORT=${BACKEND_PORT:-8000}
 
 poetry config virtualenvs.path /backend/virtualenvs
 
+poetry lock --no-update
 poetry install
 
 poetry run /backend/compilers/download.py
 poetry run /backend/libraries/download.py
 
-for reg in /backend/wine/*.reg; do
-  echo "Importing registry file $reg..."
-  regedit $reg
-done
+if command -v regedit &> /dev/null; then
+  for reg in /backend/wine/*.reg; do
+    echo "Importing registry file $reg..."
+    regedit $reg
+  done
+else
+  echo "regedit command not found. Skipping registry import."
+fi
```

Comment:
We don't need to run `regedit` if this command isn't available (e.g., when running it from ubuntu).